server/api.ts

```diff
@@ -28,17 +28,17 @@ router.use(mapRouter);
 // ----------------------
 
 const isAdmin = (user: IUser, tourney: string) => checkPermissions(user, tourney, []);
-const canViewHiddenPools = (user: IUser, tourney: string) =>
-  checkPermissions(user, tourney, [
+const canViewHiddenPools = async (user: IUser, tourney: string) =>
+  await checkPermissions(user, tourney, [
```

Comment:
btw this change is unnecessary --
the places where you have `async () => await promise` could just be `() => promise`
https://eslint.org/docs/rules/no-return-await

Reply:
Ahh didn't know return await has performance implications (was trying to make it more explicit), fixed in ea928b22de4b085c12efafe5932865fbf15e0ed3